packages/contractkit/src/wrappers/Election.ts

```diff
@@ -487,9 +484,13 @@ export class ElectionWrapper extends BaseWrapper<Election> {
    * @param address The address to retrieve VoterRewards for.
    * @param epochNumber The epoch to retrieve VoterRewards at.
    */
-  async getVoterRewards(address: Address, epochNumber: number): Promise<VoterReward[]> {
+  async getVoterRewards(
+    address: Address,
+    epochNumber: number,
+    estimate?: boolean
+  ): Promise<VoterReward[]> {
```

Comment:
I find this function signature confusion because it will mislead the reader when `estimate` is true. If it is set to true, the `epochNumber` parameter is ignored in calculation, but included in the result, which makes the result effectively inconsistent. (i.e. although is says it's with regard to epoch `epochNumber`, its actually calculated from current data. Instead I would strongly suggest having the indication to use current epoch data be to set `epochNumber` to `undefined`.

Reply:
Added missing param comment.  When `estimate` is true, the voter's share of the (actual) group voter rewards for the supplied `epochNumber` is estimated based on current votes.  This can be misleading, but only when you opt-in by supplying `estimate` = true.  Having `epochNumber` = undefined imply the latest epoch with estimation works, but I would also like to be very explicit (i.e. dedicated parameter) that the result is an estimation.

Reply:
I see your point about making it quite explicit that the returned value is an estimate. My biggest qualm here is that is that the `epochNumber` is included in the return value when using an estimate, which is misleading. I also find it misleading that the `epochNumber` argument must be supplied, but is ignored. My best suggestion here might be to split the function signature such that there is an "estimate" method that does not take the epoch reward parameter and this function which does, but does not have the estimate parameter.

Reply:
Removed the notion of estimation from contractkit by factoring out getVoterShare() from getVoterRewards().